packages/framer-motion/src/animation/animate.ts

```diff
@@ -61,12 +46,10 @@ export type AnimationOptions<V> = (Tween | Spring) &
 export function animate<V>(
     from: MotionValue<V> | V,
     to: V | V[],
-    transition: AnimationOptions<V> = {}
+    transition: Transition & AnimationPlaybackLifecycles<V> = {}
 ): AnimationPlaybackControls {
     const value = isMotionValue(from) ? from : motionValue(from)
     value.start(createMotionValueAnimation("", value, to as any, transition))
 
-    return {
-        stop: () => value.stop(),
-    }
+    return value.animation || new GroupPlaybackControls([value.animation])
```

Comment:
Why would we add the animation if it is falsy? I'd say we either always return a GroupPlaybackControls, or return an empty one when the animation doesn't exist.